my-profile-webapp/src/main/webapp/partials/emergency.html

```diff
@@ -0,0 +1,79 @@
+<div ng-if='!contact.edit || contact.edit == false'>
+    <p>{{ contact.preferredName }}</p>
+    <p ng-repeat="line in contact.addresses[0].addressLines">{{ line }}</p>
+    <p>{{ contact.addresses[0].city }}, {{ contact.addresses[0].state }} {{ contact.addresses[0].postalCode }}</p>
+    <br>
+    <div ng-repeat="phone in contact.phoneNumbers">
+        <p>{{ phone.value }} ({{ phone.type }})</p>
+    </div>
+    <br>
+    <p ng-if="contact.relationship">Relationship: {{ contact.relationship }}</p>
+    <p ng-if="contact.languages[0]">First Language: {{ contact.languages[0].value }}</p>
+    <p ng-if="contact.languages[1]">Second Language: {{ contact.languages[1].value }}</p>
```

Comment:
I agree with @apetro's approach. And yea, if we went this way, we should un-hardcode Second Language, good catch.